MAUI/TabView/Tab-Item-Customization.md

```diff
@@ -538,3 +538,28 @@ tabView.ContentTransitionDuration = 300;
 {% endhighlight %}
 
 {% endtabs %}
+
+## Image size
+
+You can customize the image size in the .NET MAUI TabView control by setting the ImageSize property.
```

Comment:
You can customize the image size in the .NET MAUI TabView control by setting the `ImageSize` property.

Reply:
Updated the content.